docs/docs/run-node/00-keyring.md

```diff
@@ -24,7 +24,7 @@ securely. Typically, an operating system's credential sub-system handles passwor
 private keys storage, and user sessions according to the user's password policies. Here
 is a list of the most popular operating systems and their respective passwords manager:
 
-* macOS (since Mac OS 8.6): [Keychain](https://support.apple.com/en-gb/guide/keychain-access/welcome/mac)
+* macOS (since macOS 8.6): [Keychain](https://support.apple.com/en-gb/guide/keychain-access/welcome/mac)
```

Comment:
```suggestion
* macOS: [Keychain](https://support.apple.com/en-gb/guide/keychain-access/welcome/mac)
```

8.6 was release 23 years ago, I think we don't need clarification here 😅 (see https://madeapple.com/mac-os-8-6/)